src/main/kotlin/net/lumalyte/lg/infrastructure/listeners/ProgressionEventListener.kt

```
@@ -65,6 +68,29 @@ class ProgressionEventListener : Listener, KoinComponent {
+        asyncTaskService.runAsyncCallback(
+            task = {
+                try {
+                    if (killer != null) {
+                        memberService.getPlayerGuilds(killer.uniqueId).forEach { gid ->
+                            leaderboardService.recordActivity(gid, ActivityType.KILL, 1)
+                        }
+                    }
+                    memberService.getPlayerGuilds(victim.uniqueId).forEach { gid ->
+                        leaderboardService.recordActivity(gid, ActivityType.DEATH, 1)
+                    }
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)